readers/cassandra/messages_test.go

```diff
@@ -215,7 +215,72 @@ func TestReadSenml(t *testing.T) {
 				Value:  v,
 			},
 			page: readers.MessagesPage{
-				Total:    uint64(len(queryMsgs)),
+				Total:    uint64(len(valueMsgs)),
+				Messages: fromSenml(valueMsgs[0:limit]),
+			},
+		},
+		"read message with value and equal comparison": {
+			chanID: chanID,
+			pageMeta: readers.PageMetadata{
+				Offset:     0,
+				Limit:      limit,
+				Value:      v,
+				Comparison: "eq",
+			},
+			page: readers.MessagesPage{
+				Total:    uint64(len(valueMsgs)),
+				Messages: fromSenml(valueMsgs[0:limit]),
+			},
+		},
+		"read message with value and lower-than comparison": {
+			chanID: chanID,
+			pageMeta: readers.PageMetadata{
+				Offset:     0,
+				Limit:      limit,
+				Value:      v + 1,
+				Comparison: "lt",
+			},
+			page: readers.MessagesPage{
+				Total:    uint64(len(valueMsgs)),
+				Messages: fromSenml(valueMsgs[0:limit]),
+			},
+		},
+		"read message with value and lower-or-equal-than comparison": {
+			chanID: chanID,
+			pageMeta: readers.PageMetadata{
+				Offset:     0,
+				Limit:      limit,
+				Value:      v + 1,
+				Comparison: "lte",
```

Comment:
I think we mentioned `le` instead of `lte`, according to ARM assembly spec.

@dusanb94 @nmarcetic opinions?